### PR TITLE
Enable recursive mop up algorithms in SBND

### PR DESCRIPTION
**Pull request by etyley**
This keeps the config here up to date with what is used in sbndcode.

### CODE REVIEW

File: settings/PandoraSettings_Neutrino_SBND.xml [hunk 1]
```diff
@@ -344,6 +344,58 @@
...
+    </MopUpAlgorithms>
+  </algorithm>
+
+  <algorithm type = "LArBdtPfoCharacterisation">
+    <TrackPfoListName>TrackParticles3D</TrackPfoListName>
+    <ShowerPfoListName>ShowerParticles3D</ShowerPfoListName>
+    <UseThreeDInformation>true</UseThreeDInformation>
+    <MvaFileName>PandoraMVAs/PandoraBdt_v08_33_00_SBND.xml</MvaFileName>
+    <MvaName>PfoCharBDT</MvaName>
+    <MvaFileNameNoChargeInfo>PandoraMVAs/PandoraBdt_v08_33_00_SBND.xml</MvaFileNameNoChargeInfo>
```
Comment (PandoraPFA) [inline settings/PandoraSettings_Neutrino_SBND.xml:382]: I note that this is the path for running in sbndcode/larsoft, and not necessarily the path you'd want to specify when running in Pandora standalone, having cloned LArMachineLearningData. We would typically modify here and so put up with a non-zero diff between the XML files for running in standalone and in sbndcode. This diff would be removal of the `PandoraMVAs/` part here (we obviously don't want to change anything about the way this runs in sbndcode/larsoft). Other diffs in our XML files are: inclusion of event reading algorithm for Pandora standalone, and whether the monitoring is on/off by default.

**Review by PandoraPFA — COMMENTED** [review]
Many thanks for adding this, Ed, to keep all the records nicely up to date, and to provide a go-to example for using the recursive pfo mop up alg.

I've added just one little comment, for quick discussion, decision and merging. Thanks again!

File: settings/PandoraSettings_Neutrino_SBND.xml (hunk 1 above)
Comment (etyley) [reply settings/PandoraSettings_Neutrino_SBND.xml]: Yes, I had done this in the other instances of where the MVAs were called previously but was a bit over zealous in my copy and pasting here. I have pushed a fix to this now.

**Comment by PandoraPFA** [comment]
Many thanks, @etyley!

**Pull request merged**